generated-codes.csv

```diff
@@ -0,0 +1,6 @@
+code,type,amount,expiration_date

```

Comment:
Please pull this file.

Reply:
Also, it would be good to add this to the `.gitignore` so it doesn't get committed back again.